pkg/client/project.go

```diff
@@ -28,12 +28,18 @@ func (c *DefaultClient) ProjectList(ctx context.Context) ([]apiv1.Project, error
 	return result.Items, nil
 }
 
-func (c *DefaultClient) ProjectCreate(ctx context.Context, name string) (*apiv1.Project, error) {
+func (c *DefaultClient) ProjectCreate(ctx context.Context, name, region string) (*apiv1.Project, error) {
 	project := &apiv1.Project{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: name,
 		},
 	}
+	if region != "" {
+		project.Spec = apiv1.ProjectSpec{
+			DefaultRegion:    region,
+			SupportedRegions: []string{region},
+		}
+	}
```

Comment:
nit: This does not introduce any issues currently so I'm fine with it going in as is. That being said - what are your thoughts on this? This doesn't completely overwrite the `project.Spec` in the case that it ever has values that are non-region related in the future. 
```suggestion
	if region != "" {
		project.Spec.DefaultRegion = region
		project.Spec.SupportedRegions = []string{region}
	}
```

Reply:
Yes, that's better.